classes/genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_reader :id, :name
+  attr_accessor :items
+
+  def initialize(id, name)
+    @id = id
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
This is requirement.
- should add self as a property of the item object (by using the correct setter from the item object)
      item.genre= self